tgctoolbox/settings.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+
+import yaml
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+BASE_FILES = [
+    os.path.join("configuration", "base.yaml"),
+    os.path.join("configuration", "local.yaml"),
+    os.path.join("configuration", "production.yaml"),
+]
+
+
+class Settings:
+    """Class to manage application settings."""

```

Comment:
we would like to have some predefined fields to be filled up.

For now it will be probably only `host` and `port`

Reply:
Sure, I assumed it will be easier to allow for more generality without specifying the extra vars, but changing this to hard coded variables is easy enough. Should I leave an optional dict for extra settings?

Reply:
Resolved in 4c8ba0f